tests/multipleBackend/backendHealthcheckResponse.js

```diff
@@ -10,15 +10,15 @@ const locConstraints = Object.keys(config.locationConstraints);
 
 describe('Healthcheck response', () => {
     it('should return no error', done => {
-        clientCheck(log, err => {
+        clientCheck(true, log, err => {
             assert.strictEqual(err, null,
                 `Expected success but got error ${err}`);
             done();
         });
     });
     it('should return result for every location constraint in ' +
     'locationConfig and at least one of every external locations', done => {
-        clientCheck(log, (err, results) => {
+        clientCheck(true, log, (err, results) => {
             locConstraints.forEach(constraint => {
```

Comment:
will you please add a test where you set clientCheck to false? will you please also unit test the utility function for the healthchecks to make sure we will get a refreshed response after a minute?  You'll have to mock the checkThisOne.healthcheck function so you are just returning different responses.